wpilib/wpilib/shuffleboard/widget.py

```diff
@@ -1,13 +1,28 @@
-# validated: 2018-11-18 EN 175c6c1f0130 edu/wpi/first/wpilibj/shuffleboard/ShuffleboardWidget.java
+# validated: 2019-01-05 TW 01d13220660c edu/wpi/first/wpilibj/shuffleboard/ShuffleboardWidget.java
 # ----------------------------------------------------------------------------
 # Copyright (c) 2018 FIRST. All Rights Reserved.
 # Open Source Software - may be modified and shared by FRC teams. The code
 # must be accompanied by the FIRST BSD license file in the root directory of
 # the project.
 # ----------------------------------------------------------------------------
+from typing import Union, TYPE_CHECKING
+
 from .component import ShuffleboardComponent
 from .container import ShuffleboardContainer
 
+if TYPE_CHECKING:
+    from typing_extensions import Protocol
```

Comment:
Pretty sure this will cause mypy errors if you don't have typing_extensions installed.

Reply:
it will. Should we add it as an optional dep like you did with camera server?

Reply:
I think we decided to get rid of this? Is that right?